README.md

```diff
@@ -104,7 +105,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+The clown hat tags that are seen in erb files are not recognized by HTML and would show up as errors.  You could remove the clown hat tags in the HTML validator so the errors don't appear.
 ```
```

Comment:
:+1: Although the quicker solution is to copy the source code from the browser and put THAT in the HTML validator.